runtime/src/main/java/io/quarkiverse/renarde/Controller.java

```diff
@@ -16,9 +16,13 @@
 import io.quarkiverse.renarde.util.RenderArgs;
 import io.quarkiverse.renarde.util.Validation;
 import io.quarkus.security.identity.SecurityIdentity;
+import io.vertx.core.http.HttpServerRequest;
 
 public class Controller {
 
+    @Inject
+    protected HttpServerRequest request;
```

Comment:
I'm not sure accessing the Vert.x request is the first thing that people will use. There's also the RESTEasy Reactive request, and I suspect it will be more useful. I'd rather stay clear of adding it to `Controller` at this point. Could you move it to the controller you're using it? We can think about this later.